src/components/TileGallery/TileGallery.story.jsx

```diff
@@ -0,0 +1,429 @@
+/* Used dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import Check from '@carbon/icons-react/lib/checkmark--filled/16';
+import Rocket from '@carbon/icons-react/lib/rocket/32';
```

Comment:
Thanks to the new entrypoint that Carbon added to `icons-react`, all icon imports can be cleaned up with this format

```diff
- import Rocket from '@carbon/icons-react/lib/rocket/32';
+ import Rocket32 from '@carbon/icons-react';
```